app/routes.py

```diff
@@ -26,32 +27,29 @@ def process_query():
     data = request.get_json()
     if not data or 'requests' not in data:
         return jsonify({"error": "Missing requests data"}), 400
-    database_type = 'metta'# data.get('database')
-    # if not database_type or database_type not in databases:
-    #     return jsonify({"error": "Invalid or missing database parameter"}), 400
+
     try:
-        db_instance = databases[database_type]
         requests = data['requests']
-        query_code = db_instance.query_Generator(requests)
+        
+        # Validate the request data before processing
+        node_map = validate_request(requests, schema_manager.schema)
+        
+        database_type = 'cypher'
```

Comment:
I suggest you use a config file to specify which database type to use, that way it is easy to alternate between Metta and cypher

Reply:
I’ve added a config file